Sources/CassandraClient/Statement.swift

```diff
@@ -201,6 +203,76 @@ extension CassandraClient {
             case float32Array([Float32])
             case doubleArray([Double])
             case stringArray([String])
+
+            case int8Int8Map([Int8: Int8])
```

Comment:
This is an API breaking which is okay since we are pre 1.0.0 but we should probably come up with a way to introduce new types without breaking API. This doesn't need to be solved in this PR though